pkg/rootless/rootless_linux.go

```diff
@@ -229,6 +255,15 @@ func readUserNsFd(fd uintptr) (string, error) {
 	return readUserNs(filepath.Join("/proc/self/fd", fmt.Sprintf("%d", fd)))
 }
 
+func getOwner(fd uintptr) (uintptr, error) {
+	const nsGetUserns = 0xb701
```

Comment:
question: is this kernel 4.9 feature is back ported to RHEL 7.6?

Reply:
I am not sure if rootless containers are going to be fully supported for RHEL 7.6 (including this change in the kernel)